### PR TITLE
[ru] Localize `function-as-a-service`

### LINKED ISSUE #3060: [ru] Localize `Function as a Service (FaaS)`
### Description

This Issue is to localize [`Function as a Service (FaaS)`](https://github.com/cncf/glossary/blob/main/content/en/function-as-a-service.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by kirkonru**
### Describe your changes

Localize `function-as-a-service`

### Related issue number or link (ex: `resolves #issue-number`)

resolves #3060 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/function-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Функция-как-сервис (FaaS)
```
Comment (shurup) [inline content/ru/function-as-a-service.md:2]: I have the same question regarding hyphens here :sweat_smile: I'd better remove them.

File: content/ru/function-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+status: Completed
+category: Technology
+tags: ["infrastructure", "", ""]
+---
+
+Функция-как-сервис (Function as a Service, FaaS) — модель облачных вычислений, в которой функции, вызванные событиями, выполняются на сторонней платформе. Ее главный плюс — автоматическое масштабирование, не требующее ручного вмешательства.
```
Comment (shurup) [inline content/ru/function-as-a-service.md:8]:
```suggestion
Функция-как-сервис (Function as a Service, FaaS) — модель облачных вычислений, которая предлагает платформу для выполнения функций, инициированных событиями. Она обеспечивает автоматическое масштабирование, не требующее ручного вмешательства.
```

File: content/ru/function-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+В сущности FaaS позволяет развёртывать отдельные функции, которые активируются в ответ на определенные события, некоторое (короткое) время работают и отключаются. Тем самым гарантируется, что ресурсы не тратятся впустую.
+Модель поддерживает [автоматическое масштабирование](/ru/auto-scaling/), позволяя запускать экземпляр функции по запросу и завершать его после выполнения, подчеркивая stateless-природу.
```
Comment (shurup) [inline content/ru/function-as-a-service.md:10]: However, this [original] "stateless" notice doesn't seem to be really helpful here…
```suggestion
Модель поддерживает [автоматическое масштабирование](/ru/auto-scaling/), позволяя запускать экземпляр функции по запросу и завершать его после выполнения, что соответствует его stateless-природе.
```

File: content/ru/function-as-a-service.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+FaaS-платформы реализуют подход к тарификации по принципу "плати за фактическое использование": когда функция не работает, она не потребляет ресурсы, экономя деньги. Этим они отличаются от других моделей, таких как [Платформа-как-услуга](/platform-as-a-service/) (Platform as a Service, PaaS), которые предполагают постоянную доступность ресурсов.
+
+## Какую проблему решает
+
+Традиционно компании предпочитали работать с собственными центрами обработки данных, что требовало значительных инвестиций в оборудование, программное обеспечение и персонал.
+Такой подход означал, что ЦОД должен был проектироваться под пиковый спрос — т.е. во все остальное время его ресурсы использовались лишь частично.
```
Comment (shurup) [inline content/ru/function-as-a-service.md:16]:
```suggestion
Такой подход означал, что ЦОД должен был проектироваться под пиковый спрос, а в остальное время его ресурсы использовались лишь частично.
```

File: content/ru/function-as-a-service.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Кроме того, стремительное развитие бизнеса могло опередить возможности ИТ, приводя к операционой неэффективности.
```
Comment (shurup) [inline content/ru/function-as-a-service.md:17]:
```suggestion
Кроме того, стремительное развитие бизнеса могло опередить возможности ИТ и привести к операционной неэффективности.
```

File: content/ru/function-as-a-service.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+С другой стороны, модели вида [Инфраструктура-как-услуга](/infrastructure-as-a-service/) (Infrastructure-as-a-Service, IaaS), хотя и предлагают облачные решения, все же возлагают бремя масштабирования ресурсов на пользователя, требуя оплаты за постоянную доступность сервера независимо от фактического использования.
+
+## Как именно решает проблему
+
+FaaS предоставляет разработчикам [абстракцию](/ru/abstraction/) для запуска веб-приложений в ответ на события, избавляя их от необходимости управлять серверной инфраструктурой.
+Например, загрузка файла может запустить кастомный код, который перекодирует файл в различные форматы.
+Инфраструктура FaaS автоматически регулирует ресурсы в зависимости от спроса, освобождая разработчиков от необходимости писать код с учетом [масштабируемости](/scalability/) и связанных с этим сложностей.
+Плата взимается только за время вычислений — когда функции неактивны, деньги не списываются.
```
Comment (shurup) [inline content/ru/function-as-a-service.md:25]:
```suggestion
Плата взимается только за время вычислений: когда функции неактивны, деньги не списываются.
```

**Review by shurup — COMMENTED** [review]
Just a few suggestions to consider :pray:

**Comment by shurup** [comment]
Will we prefer "Функция как сервис" instead of the currently used "Функция-как-сервис"?

File: content/ru/function-as-a-service.md (hunk 1 above)
Comment (kirkonru) [inline content/ru/function-as-a-service.md:2]:
```suggestion
title: Функция как сервис (FaaS)
```

**Review by shurup — APPROVED** [review]
Thanks! LGTM

**Pull request merged**